opennlp-tools/src/main/java/opennlp/tools/util/StringUtil.java

```diff
@@ -79,6 +81,30 @@ public static String toLowerCase(CharSequence string) {
     return new String(cp, 0, cp.length);
   }
 
+  public static CharBuffer toLowerCaseCharBuffer(CharSequence sequence) {
+    CharBuffer result = CharBuffer.allocate(sequence.length());
+    for (int cp : sequence.codePoints().toArray()) {
+      for (char c : Character.toChars(Character.toLowerCase(cp))) {
+        result.append(c);
+      }
+    }
+    result.clear();
+    return result;
+  }
+
+  /*
+  public static CharBuffer toLowerCaseCharBuffer(CharSequence string) {
```

Comment:
Old code?

Reply:
Will do / beautify once PR passes thorough testing from the community or build servers.